doc/operator-upgrade.md

```diff
@@ -115,6 +115,18 @@ The upgrade to Operator 0.5.0 can be triggered using the following command
 helm upgrade <operator release name> <location of 0.5.0 charts> --set webhookCert.crt=<tls.crt> --set webhookCert.generate=false --set webhookCert.certName=<cert-name> --set webhookCert.secretName=<secret-name>
```

Comment:
Should it be obvious here that the `helm upgrade` command should be run three times (once for each of the operators), or are those instructions in each operator's GitHub repo?

Reply:
upgrade of zookeeper operator is not required.. and its already mentioned that installation of bookkeeper operator is a prerequite to the upgrade